examples/Playground.hs

```diff
@@ -1,3 +1,5 @@
+{-# LANGUAGE NoImplicitPrelude #-}
```

Comment:
You don't need `NoImplicitPrelude` in this file because executable target `play-log` already contains `NoImplicitPrelude` inside `default-extensions`. As I see now, `NoImplicitPrelude` is also inside `default-extensions` for `library`. So I think we can remove `{-# LANGUAGE NoImplicitPrelude #-}` from every file and nothing should break.